docs/readmes/orc8r/deploy_build.md

```diff
@@ -107,9 +132,10 @@ The valid deployment type options are
 Run the package script
 
 ```bash
-$ ${MAGMA_ROOT}/orc8r/tools/helm/package.sh -d fwa  # or chosen deployment type
+$ ${MAGMA_ROOT}/orc8r/tools/helm/package.sh -d fwa [-v 1.5] # or chosen deployment type
```

Comment:
Partners copy+paste these commands, they will get confused by the `[-v 1.5]` syntax. Either remove it here, or add a commented-out version of the command below showing the option